index.php

```diff
@@ -20,7 +20,7 @@
         <!-- event banner: uncomment this section when we have an event or promotion -->
         <div class="section section-announcement">
           <div class='wrapper'>
-            <p style='font-size:2em'><a href='/14/'>New (15 Dec 2020): Keyman 14.0 is now in beta!</a></p>
+            <p style='font-size:2em'><a href='/14/'>New (22 Mar 2021): Keyman 14.0 is now available to download!</a></p>
```

Comment:
Do you want this 24 Mar 2021 also?

Reply:
Yeah or perhaps 23 if I can get it done today. Waiting on other PRs to be approved and merged and I'll kick off a beta build immediately after that